### PR TITLE
Release machine_stats on PyPI

### PR DESCRIPTION
**Pull request by pshevtsov**
This PR prepares `machine_stats` for Unix-like systems to be released on the [Python Package Index](https://pypi.org/):

- It provided better package structure
- It makes it still possible to use `pipenv` (but mainly for development)

Manually released version is available at https://pypi.org/project/machine-stats/ so it is now possible to install the standard way executing

```
pip install machine-stats
```

TODO:

- [x] Automate new versions releases using GitHub actions (implemented in https://github.com/tidalmigrations/machine_stats/pull/29/commits/29a58902f49b7b21e9735c0e709762bc7d360abe)
- [ ] Update documentation

### CODE REVIEW

File: unix/setup.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+from distutils.core import setup
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setup(
```
Comment (jmariomejiap) [inline unix/setup.py:6]: This is really great, that package description looks awesome

**Review by jmariomejiap — APPROVED** [review]
Very nice Python code @pshevtsov 🐍 🧙

**Review by prees1 — COMMENTED** [review]
Looks great @pshevtsov 🎉 

I take it we will complete these two todos before this is merged, right?
TODO: Automate new versions releases using GitHub actions.
TODO: Update documentation

I take it the existing installation instructions won't work with this setup. Also in the updated documentation steps, I think worthwhile to mention, very briefly, how to install it manually or 'offline'. As I think it means going to the python package website and downloading it, right? and then some command? currently, it's mostly clear at least to me, that with the github approach you put the repo where github would clone it too.

**Comment by pshevtsov** [comment]
> I take it we will complete these two todos before this is merged, right?
> TODO: Automate new versions releases using GitHub actions.
> TODO: Update documentation

Actually, no. I think it's OK to merge now and implement those todos later:

1. It's pretty easy to release new versions manually. I did so for three times already to investigate on some issues and check for fixes.
2. The existing documentation still works, i.e. it's still possible to use `pipenv` for installation and running.

However, the above items are pretty easy to implement.

**Comment by prees1** [comment]
Ok great! I think you are set to merge then if the documentation is still functional.

**Comment by pshevtsov** [comment]
> Ok great! I think you are set to merge then if the documentation is still functional.

```console
$ git clone https://github.com/tidalmigrations/machine_stats.git
...
$ cd machine_stats/
$ git checkout pypi 
...
$ cd unix/
$ pipenv install
...
$ pipenv run stats --help
usage: machine_stats [-h] [FILE [FILE ...]]

positional arguments:
  FILE        inventory file (default 'hosts')

optional arguments:
  -h, --help  show this help message and exit
```

**Pull request merged**